src/System.Reflection.MetadataLoadContext/src/Resources/Strings.resx

```diff
@@ -0,0 +1,273 @@
+    <value>Must be an array type.</value>
+  </data>
+  <data name="Argument_InvalidEnum" xml:space="preserve">
+    <value>The Enum type should contain one and only one instance field.</value>
+  </data>
+  <data name="Arg_AmbiguousMatchException" xml:space="preserve">
+    <value>Ambiguous match found.</value>
+  </data>
+  <data name="Arg_EmptyArray" xml:space="preserve">
+    <value>Array may not be empty.</value>
+  </data>
+  <data name="Arg_EnumLitValueNotFound" xml:space="preserve">
+    <value>Literal value was not found.</value>
+  </data>
+  <data name="Arg_HTCapacityOverflow" xml:space="preserve">
+    <value>Hashtable's capacity overflowed and went negative. Check load factor, capacity and the current size of the table.</value>
+  </data>
+  <data name="Arg_MustBeEnum" xml:space="preserve">
+    <value>Type provided must be an Enum.</value>
+  </data>
+  <data name="Arg_MustBeType" xml:space="preserve">
+    <value>Type must be a type provided by the MetadataLoadContext.</value>
+  </data>
+  <data name="Arg_NotGenericMethodDefinition" xml:space="preserve">
+    <value>{0} is not a GenericMethodDefinition. MakeGenericMethod may only be called on a method for which MethodBase.IsGenericMethodDefinition is true.</value>
+  </data>
+  <data name="Arg_NotGenericParameter" xml:space="preserve">
+    <value>Method may only be called on a Type for which Type.IsGenericParameter is true.</value>
+  </data>
+  <data name="Arg_NotGenericTypeDefinition" xml:space="preserve">
+    <value>{0} is not a GenericTypeDefinition. MakeGenericType may only be called on a type for which Type.IsGenericTypeDefinition is true.</value>
+  </data>
+  <data name="Arg_ReflectionOnlyCA" xml:space="preserve">
+    <value>The requested operation is invalid on objects loaded by a MetadataLoadContext. Use CustomAttributeData instead.</value>
+  </data>
+  <data name="Arg_ReflectionOnlyIllegal" xml:space="preserve">
+    <value>The requested operation is invalid on objects loaded by a MetadataLoadContext.</value>
+  </data>
+  <data name="Arg_ReflectionOnlyInvoke" xml:space="preserve">
+    <value>It is illegal to invoke a method on a Type loaded by a MetadataLoadContext.</value>
+  </data>
+  <data name="Arg_ReflectionOnlyParameterDefaultValue" xml:space="preserve">
+    <value>It is illegal to request the default value on a ParameterInfo loaded by a MetadataLoadContext. Use RawDefaultValue instead.</value>
+  </data>
+  <data name="BadImageFormat_TypeRefBadScopeType" xml:space="preserve">
+    <value>Assembly '{0}' contains a type reference 0x{1} that contains an invalid scope token.</value>
+  </data>
+  <data name="BadImageFormat_TypeRefModuleNotInManifest" xml:space="preserve">
+    <value>Assembly '{0}' contains a type reference 0x{1} that references a module not in the manifest.</value>
+  </data>
+  <data name="CoreTypeNotFound" xml:space="preserve">
+    <value>Could not find core type '{0}'</value>
```

Comment:
Nit, period

Reply:
Added periods to each resource string that didn't have one.